go/enclave/evm/evm_facade.go

```diff
@@ -94,9 +77,7 @@ func ExecuteOffChainCall(from gethcommon.Address, to gethcommon.Address, data []
 
 func initParams(rollupResolver db.RollupResolver) (*ObscuroChainContext, vm.Config, *gethcore.GasPool) {
 	chain := &ObscuroChainContext{rollupResolver: rollupResolver}
-	vmCfg := vm.Config{
-		NoBaseFee: true,
```

Comment:
since we reuse this for both eth_call and normal transactions, you have to pass this as a parameter.
In the execute transaction it should be false, and in the offchain tx it should be true.

Reply:
Thanks, fixed.